doc/sphinx/source/windows.rst

```diff
@@ -0,0 +1,71 @@
+
+Windows support
+-----------------
+
+
+It is possible to build & use PoCL on Windows using MinGW (MSVC port is in progress). However, the setup requires a number of steps.
```

Comment:
```suggestion
It is possible to build & use PoCL on Windows using MinGW. However, the setup requires a number of steps.
```
No need to mention MSVC here, we will add the section after the MSVC support is brought back.